src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -75,7 +73,7 @@
   </Target>
 
   <Target Name="RunPublishedCrossgen" AfterTargets="PublishCrossgen"
-          Condition="'$(TargetOS)' == '$(HostOS)' and '$(TargetArchitecture)' == '$(BuildArchitecture)'">
+          Condition="'$(TargetOS)' == '$(HostOS)' and '$(TargetArchitecture)' == '$(BuildArchitecture)' and '$(TargetsLinuxMusl)' != 'true'">
```

Comment:
```suggestion
          Condition="'$(TargetOS)' == '$(HostOS)' and '$(TargetArchitecture)' == '$(BuildArchitecture)' and '$(CrossBuild)' != 'true'">
```

This way if we are building on linux-musl-x64, this target will run and only skip if we are cross-building for linux-musl-x64 on linux-x64.